web/db/migrations/20260724140000_iroh_rekey_bindings/migration.sql

```
@@ -0,0 +1,58 @@
+  SELECT
+    id,
+    user_id,
+    row_number() OVER (
+      PARTITION BY user_id, device_uuid, tag
+      ORDER BY last_seen_at DESC, registered_at DESC, id DESC
+    ) AS rn
+  FROM iroh_endpoint_bindings
+  WHERE revoked_at IS NULL
+),
+losers AS (
+  SELECT id FROM ranked WHERE rn > 1
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)